XenModel/Messages.resx

```diff
@@ -1874,12 +1874,18 @@ Archive destination: {0}</value>
   <data name="ASSIGNED_PROTECTION_POLICY" xml:space="preserve">
     <value>Protection policy assigned.</value>
   </data>
+  <data name="ASSIGNED_VMSS_POLICY" xml:space="preserve">
+    <value>Snapshot Schedule assigned.</value>

```

Comment:
There's a lot of inconsistency whether we write Snapshot Schedule with or without upper case letters. I recommend standardising on lower case.